Mvc.sln

```diff
@@ -993,5 +1003,7 @@ Global
 		{EA34877F-1AC1-42B7-B4E6-15A093F40CAE} = {DAAE4C74-D06F-4874-A166-33305D2643CE}
 		{DAB1252D-577C-4912-98BE-1A812BF83F86} = {DAAE4C74-D06F-4874-A166-33305D2643CE}
 		{FCFE6024-2720-49B4-8257-9DBC6114F0F1} = {DAAE4C74-D06F-4874-A166-33305D2643CE}
+		{72AF67AF-D523-4321-AF0F-2287117E67A0} = {16703B76-C9F7-4C75-AE6C-53D92E308E3C}
```

Comment:
`{72AF67AF-D523-4321-AF0F-2287117E67A0}` project ref should be deleted here too.